skimage/filters/thresholding.py

```diff
@@ -881,3 +881,42 @@ def threshold_sauvola(image, window_size=15, k=0.2, r=None):
         r = 0.5 * (imax - imin)
     m, s = _mean_std(image, window_size)
     return m * (1 + k * ((s / r) - 1))
+
+
+def apply_hysteresis_threshold(image, low, high):
```

Comment:
@soupault yes, it is reasonable, and actually it would just work, *if* you can guarantee that `low <= high`. Maybe I'll add a call to np.clip and add the new types to the docstring.

@foloumi I believe that `hysteresis_threshold` would add to the confusion that this is an inconsistently-named thresholding function. So I'd like to keep the `apply_` in. (Autocomplete fixes the length problem in most scenarios.) If anyone has suggestions for a more concise name *that clearly indicates* the downstream nature of this function, I am open to suggestions.